index.js

```diff
@@ -64,17 +65,29 @@ const ignored = [
   {
     property: "display",
     value:
-      "/^table-(row|row-group|column|column-group|header-group|footer-group|caption).*$/",
+      "/^table-(row|row-group|column|column-group|header-group|footer-group)$/",
```

Comment:
> padding has no effect on this element since it has a table-row-group display type
padding has no effect on this element since it has a table-header-group display type
padding has no effect on this element since it has a table-footer-group display type
padding has no effect on this element since it has a table-row display type
padding has no effect on this element since it has a table-column-group display type
padding has no effect on this element since it has a table-column display type

https://github.com/captainbrosset/useless-css-properties/blob/master/rules.md#margin-and-padding-properties-on-tables

---

> padding: applies to all elements, except table-row-group, table-header-group, table-footer-group, table-row, table-column-group and table-column.

https://developer.mozilla.org/en-US/docs/Web/CSS/padding

Reply:
There is a small difference with MDN vs useless-css-properties list with padding. I suspect that MDN just does not list all the ignored properties and that's why there's a difference.